src/site/content/en/blog/file-handling/index.md

```diff
@@ -27,7 +27,7 @@ origin_trial:
 Now that web apps are [capable of reading and writing files](/file-system-access/), the next logical
 step is to let developers declare these very web apps as file handlers for the files their apps can
 create and process. The File Handling API allows you to do exactly this.
-After registering a text editor app as a file handler, you can right-click a `.txt` file on macOS
+After registering a text editor app as a file handler and after installing it, you can right-click a `.txt` file on macOS
```

Comment:
Is it even possible to register an app without installing it? A reader of this article will be processing much new information. Don't expect them to make logical connections at the same time. If there's a concern here, spell it out.